cypress/fixtures/flows/dashboard-number-input.json

```diff
@@ -0,0 +1,466 @@
+        "icon": "numeric",
+        "iconPosition": "left",
+        "iconInnerPosition": "inside",
+        "x": 980,
+        "y": 760,
+        "wires": [
+            []
+        ]
+    },
+    {
+        "id": "696f064e3552fe7d",
+        "type": "ui-number-input",
+        "z": "f4f554a38c3ce158",
+        "g": "1f4fd97829933c1b",
+        "group": "5107b8a93a5ac068",
+        "name": "",
+        "label": "number global store",
+        "order": 0,
+        "width": 0,
+        "height": 0,
+        "topic": "topic",
+        "topicType": "msg",
+        "min": 0,
+        "max": 10,
+        "step": 1,
+        "tooltip": "Tooltip Text",
+        "delay": 300,
+        "passthru": true,
+        "className": "",
+        "clearable": true,
+        "icon": "",
+        "iconPosition": "left",
+        "iconInnerPosition": "inside",
+        "x": 620,
+        "y": 580,
+        "wires": [
+            [
+                "d92b128565cf0da0"
+            ]
+        ]
+    },
+    {
+        "id": "05b923de3f2d2970",
+        "type": "ui-group",
+        "name": "Page1 Group2",
+        "page": "9849a6ac1bf4073a",
+        "width": "6",
+        "height": "1",
+        "order": 4,
+        "showTitle": true,
+        "className": "",
+        "visible": "true",
+        "disabled": "false"
+    },
+    {
+        "id": "730d0fcc2b4628ba",
+        "type": "ui-group",
+        "name": "Page1 Group3",
+        "page": "9849a6ac1bf4073a",
+        "width": "6",
+        "height": "1",
+        "order": 5,
+        "showTitle": true,
+        "className": "",
+        "visible": "true",
+        "disabled": "false"
+    },
+    {
+        "id": "6e46363f590502ee",
+        "type": "ui-group",
+        "name": "Page1 Group4",
+        "page": "9849a6ac1bf4073a",
+        "width": "6",
+        "height": "1",
+        "order": 6,
+        "showTitle": true,
+        "className": "",
+        "visible": "true",
+        "disabled": "false"
+    },
+    {
+        "id": "5107b8a93a5ac068",
+        "type": "ui-group",
+        "name": "Page1 Group1",
+        "page": "9849a6ac1bf4073a",
+        "width": "6",
+        "height": "1",
+        "order": 3,
+        "showTitle": true,
+        "className": "",
+        "visible": "true",
+        "disabled": "false"
+    },
+    {
+        "id": "9849a6ac1bf4073a",
+        "type": "ui-page",
+        "name": "Page1",
+        "ui": "a171c8195c1b8e57",
+        "path": "/page1",
+        "icon": "numeric-1",
+        "layout": "grid",
+        "theme": "9d8bfd7e0d216779",
+        "order": 1,
+        "className": "",
+        "visible": true,
+        "disabled": false
+    },
+    {
+        "id": "a171c8195c1b8e57",
+        "type": "ui-base",
+        "name": "My Dashboard",
+        "path": "/dashboard",
+        "includeClientData": true,
+        "acceptsClientConfig": [
+            "ui-control",
+            "ui-gauge",
+            "ui-chart",
+            "ui-form",
```

Comment:
You're importing a completely new `u-base` here - use the same `ui-base` element as the other tests please - it'll prevent a lot of these clashes I think we're seeing

Reply:
Used the existing `ui-base` but still crashes randomly